Somesh/HTML/Assignment6/que1Assign6.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Question1Assignment6</title>
+        <style>
+            h1,
+            h2 {
+                background-color: rgb(0, 163, 109);
+                text-align: center;
+                color: white;
+                font-family: Poppins;
+            }
+        </style>
+    </head>
+    <body>
+        <h1>MI-11 Intern Html Assignent</h1>
+        <br />
+        <br />
+        <hr />
+        <h2>Personal Details</h2>
+        <label for="F-name">First name</label>
+        <input type="text" placeholder="First Name" />
+        <br />
+        <br />
+        <label for="L-name">Lasr name</label>
+        <input type="text" placeholder="Last Name" />
+        <br />
+        <br />
+        Gender
+        <input type="radio" name="gender" />
+        <input type="radio" name="gender" />
+        <input type="radio" name="gender" />
+        <br />
+        <br />
+        Age
+        <input type="radio" name="age" />less than 18
+        <input type="radio" name="age" />18-25
+        <input type="radio" name="age" />25-40
+        <br />
+        <br />
+        <h2>Academic Details</h2>
+        Skill Set
+        <input type="checkbox" name="skill" id="" /> HTML
+        <input type="checkbox" name="skill" id="" /> JavaScript
+        <input type="checkbox" name="skill" id="" /> Java
+        <input type="checkbox" name="skill" id="" /> Oracle
+    </body>
+</html>
```

Comment:
leave one extra line after closing html tag

Reply:
i have but i tkink because of gihub code standards i wont show here